test/directory_watcher/windows_test.dart

```diff
@@ -4,19 +4,19 @@
 
 @TestOn('windows')
 
-import 'package:scheduled_test/scheduled_test.dart';
+import 'package:test/test.dart';
 import 'package:watcher/src/directory_watcher/windows.dart';
 import 'package:watcher/watcher.dart';
 
-import 'shared.dart';
+// import 'shared.dart';
 import '../utils.dart';
 
 void main() {
   watcherFactory = (dir) => new WindowsDirectoryWatcher(dir);
 
-  setUp(createSandbox);
-
-  sharedTests();
+  // TODO(grouma) - renable when https://github.com/dart-lang/sdk/issues/31760
+  // is resolved.
+  // sharedTests();
```

Comment:
Skip tests rather than commenting them out.

Reply:
It would probably be cleaner to add the group here, rather than manually forwarding parameters to `sharedTests()` and adding indentation and prefixes to every test (which also makes the diff a lot less clear).